plugins/inventory/flightctl.py

```
@@ -422,10 +426,29 @@ def _add_to_group(self, group_name: str, device_id: str):
 
 
 # ---------------------- Custom context manager ------------------
+def _set_org_id_query_param(client: ApiClient, organization: str) -> None:
+    """Inject the optional Flight Control org selector as a query parameter."""
+    original_param_serialize = client.param_serialize
+
+    def param_serialize_with_org_id(*args, **kwargs):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `param_serialize_with_org_id`

(ANN202)